ads/mobusi.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript} from '../3p/3p';
+
+export function mobusi(global, data) {
+  const adsrc = data.src;
+  const timestamp = (Date.now() % 2147483648) + Math.random();
+  if (typeof adsrc != 'undefined') {
+    writeScript(global, encodeURIComponent(adsrc + '&lgid=' + timestamp));
```

Comment:
As chained to join the same URL as adsrc, I think the right thing is that it also encode